### ISSUE TITLE
Error on startup, uncaught exception

### ISSUE THREAD

**Issue by andpeterson** [chrisknepper/android-messages-desktop#184]
Windows error when launching Android Messages on Windows 10. Started today for no reason. 
![image](https://user-images.githubusercontent.com/13080580/59144916-8d931280-8991-11e9-919a-2c1847fe7074.png)

**Comment by Archmonger** [comment]
Confirmed this issue occurs frequently on the latest release.

**Comment by nelsonni** [comment]
Confirmed to be occurring on my Windows 10 system as well:
```
Edition:	Windows 10 Enterprise
Version:	1903
OS build:	18362.175
```
Possibly an issue with registering incoming messages for display in the Windows 10 Notification Center?

**Comment by Archmonger** [comment]
I don't believe it has to do with push notifications. Notifications do not cause any crashing behavior. Crashes only occur when opening the GUI.

**Comment by nelsonni** [comment]
This error specifically occurs when I have Android Messages minimized to the task bar and new messages are received. I get one for each new message that arrives, and have to click OK on each of them in turn. Which is why I originally attributed this issue to notifications.

**Comment by kcsodetz** [comment]
I am also having this issue, any word on what may be causing it?

**Comment by johnjanney** [comment]
I also get this error message when a new message comes in. 

UPDATE: I uninstalled and reinstalled. It seems to fix it. Hopefully, the error won't return. If it does, I'm assuming the only fix, for now, is to uninstall and reinstall.

**Comment by palinkas-jo-reggelt** [comment]
This happens randomly for me. As mentioned above, usually (always?) when minimized to tray. It works for a few days and then I get this error that requires ending from task manager to end/clear.
